py/desispec/qa/qa_exposure.py

```diff
@@ -85,7 +85,11 @@ def fluxcalib(self, outfil):
 
     def load_qa_data(self, remove=True):
         """ Load the QA data files for a given exposure (currently yaml)
+        Args:
+            remove: bool, optional
+              Remove frame files
```

Comment:
Presumably this means remove the QA frame files, not the frame files themselves.